src/ceph.in

```diff
@@ -27,6 +27,9 @@ import subprocess
 import time
 import platform
 
+reload(sys)
+sys.setdefaultencoding('utf8')
```

Comment:
could you pinpoint the place where we do the decoding a utf-8 string using the default codec? i don't really think it's advisable to reset the default encoding this way.